helm/cluster-cloud-director/values.schema.json

```diff
@@ -149,49 +95,76 @@
                 }
             }
         },
-        "kubeadm": {
+        "defaultStorageClass": {
```

Comment:
@vxav can you check `required` parts for these objects? Many of them are missing.

Reply:
See https://github.com/giantswarm/cluster-cloud-director/commit/239c5a3dcb512e910c8c899fc2e3cecd9502a56d